vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -7,11 +7,50 @@ sidebar_position: 1
 import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
 
 import ApiKey from '@site/platform/api/_partials/resources/config/external/platform/apiKey.mdx'
+import Flow, { Step } from '@site/src/components/Flow';
 
 <ProAdmonition/>
 
+<!--vale off-->
+
 This provides a way to connect directly to the vCluster Platform.
 
+## Connect vCluster to the platform 
+
+You can connect virtual clusters to the platform using a managed API key. 
```

Comment:
In the UI of the platform this is called access key instead of API key. 

We should add in steps/docs link of how to get it.

Reply:
This is a good idea for non-versioned partial, because this is used in multiple places.